tests/unit_tests/CMakeLists.txt

```diff
@@ -30,7 +30,6 @@ set(unit_tests_sources
   account.cpp
   apply_permutation.cpp
   address_from_url.cpp
-  ban.cpp
```

Comment:
Why ?

Reply:
Renamed it into `node_server.cpp` cause i'm using `test_core` defined there.

Reply:
OK. I'd have moved the test_core in a separate file and used it in both, but that works too. This can be undone later if the file starts to accrete too much stuff.